webhooks-extension/DEVELOPMENT.md

```diff
@@ -4,6 +4,8 @@ Involved development scripts and instructions can be found in the [development i
 
 ## UI development
 
+Use `npm ci` when installing from source to install with a clean slate of dependencies, if a node_modules folder is already present in the project root it will be automatically removed before install.
```

Comment:
I would use "clean set" rather than "clean slate" to simplify any translation issues